proto/viam/app/datasync/v1/data_sync.proto

```diff
@@ -51,14 +50,15 @@ message UploadMetadata {
   string part_id = 1;
   string component_type = 2;
   string component_name = 3;
-  string component_model = 4;
   string method_name = 5;
   DataType type = 6;
   string file_name = 7;
   map<string, google.protobuf.Any> method_parameters = 8;
   string file_extension = 9;
   repeated string tags = 10;
   string session_id = 11;
```

Comment:
Should session_id be removed too? I don't think it's ever been used. I know not in the scope though - so maybe should save for some other time